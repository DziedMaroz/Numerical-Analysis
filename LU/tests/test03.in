1 4 5 2 20
1 2 3 1 11
7 9 10 2 40
2 9 8 3 27
