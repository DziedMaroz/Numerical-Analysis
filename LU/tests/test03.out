16.667 -1.333 -14.667 41.000 